### PR TITLE
Builds Out Old Classes, Removes RNG

### PR DESCRIPTION
**Pull request by Stagrawl**
Bunch of small changes, main focus is to build out some of the older less developed classes and remove horrible skill RNG from others. While I was in there, also cleans up some loadout jank.

This isn't really a balance passover, mostly a style and good practice run through. It does slightly touch balance, but generally for the better.

Expands:

- Bath Maid - Whip Skill + Few Bump Ups
- Barkeep - Sword Skill + Better Hand-to-Hand Utility
- Mage Apprentice - Basically turns this into a mage. (Until Rituals are implemented)

Adjusts (Removal of RNG or Cleanups, tiny number adjustments.)

- Blacksmith
- Mason
- Merchant
-  Mortician
- Jester
- Magician
- Acolyte
- Carpenter
- Miner
- Woodcutter

Does nothing to slots.

### CODE REVIEW

**Comment by Stagrawl** [comment]
> Can you give this a name. The fact that its WIP is implied by the PR being a draft you smartass

No I actually want it to be as obtuse as possible for anyone who doesn’t have the determination to click view files. 

I’ll name it properly at some point.

Intended.

File: code/modules/jobs/job_types/roguetown/yeomen/mason.dm [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
-		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, rand(1,3), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/crafting, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/carpentry, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/masonry, rand(4,5), TRUE)
-		H.mind.adjust_skillrank(/datum/skill/craft/engineering, rand(2,4), TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/mining, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/axes, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/maces, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/lumberjacking, 2, TRUE)
```
Comment (Hmeister-real) [inline code/modules/jobs/job_types/roguetown/yeomen/mason.dm:26]: you gave them lumberjacking instead of mining

Comment (Stagrawl) [reply code/modules/jobs/job_types/roguetown/yeomen/mason.dm:26]: ?

Comment (Hmeister-real) [reply code/modules/jobs/job_types/roguetown/yeomen/mason.dm:26]: its a mason no? Rockworker + miner

**Pull request merged**